docs/components/modeler/web-modeler/save-as-element-templates.md

```diff
@@ -0,0 +1,161 @@
+---
+id: save-as-element-templates
+title: Save tasks as element templates
+description: Learn how to save any task as a reusable element template in Camunda Web Modeler.
+---
+
+import BusinessRuleTaskImg1 from './img/save-as-template/1-business-rule-task.png';
+import SaveAsTemplateButtonImg2 from './img/save-as-template/2-save-as-template-button.png';
+import TemplateSavedImg3 from './img/save-as-template/3-template-saved.png';
+import EditTemplateImg4 from './img/save-as-template/4-edit-template.png';
+import FinalizedTemplateImg5 from './img/save-as-template/5-finalized-template.png';
+import PublishTemplateImg6 from './img/save-as-template/6-publish-template.png';
+import ApplyTemplateImg7 from './img/save-as-template/7-apply-template.png';
+
+# Save tasks as element templates
+
+Element templates allow you to create reusable, configurable building blocks that can be shared across your organization.
+With the "Save as template" feature, you can convert any supported task into an element template.
+
+## Supported task types
+
+The "Save as template" feature is available for the following task types:
+
+- Service task
+- User task
+- Send task
+- Receive task
+- Business rule task
+- Script task
+- Manual task
+- Call activity
+
+Note that you cannot create a template for the [undefined task type](../../bpmn/undefined-tasks).
+
+## Creating an element template from a task
+
+You can save any supported task as an element template directly from the properties panel in Web Modeler.
+Configure your task with the desired properties, save it as a template, and optionally customize it further in the template editor.
+
+### Prerequisites
+
+Before saving a task as a template:
+
+- Ensure the task is properly configured with the desired properties
+- Resolve any validation errors on the task (the "Save as template" button is disabled if there are active errors)
+
+### Step 1: Configure your task
+
+First, configure your task with all the necessary properties that you want to include in your template.
+The specific configuration will depend on your task type and use case.
+
+In this example, we'll configure a Business Rule Task for fraud detection:
+
+1. Select a task in your BPMN diagram.
+2. Configure the task with the properties you need. For our example, we will set up a Business Rule Task by setting:
+   - Implementation: Choose the implementation type (for example, DMN decision)
+   - Called decision: Reference the decision to be invoked
+   - Binding type: Select the version binding strategy
```

Comment:
[❓ ] Would it make sense to link this to the documentation related to binding types?